tests/test-valid-json/test.yaml

```diff
@@ -0,0 +1,17 @@
+requires:
+  min-version: 8
+
+command: |
```

Comment:
can this be replaced by something like
```
args:
- -T
pcap: false
```
?

Reply:
e.g. https://github.com/OISF/suricata-verify/blob/master/tests/content-incomplete-hex-t-version-7-plus/test.yaml

Reply:
Really cleaner indeed.

Reply:
I've forced pushed but should I do another MR ?

Reply:
I think force push is fine here for a case like this.